### PR TITLE
change year to 2024

### PR DESCRIPTION
**Pull request by khaledatia25**
I have edited all occurrence of 
- `Copyright (c) 2023 Orange` to `Copyright (c) 2024 Orange`
- `SPDX-FileCopyrightText: Copyright (c) 2022-2023 Orange` to `SPDX-FileCopyrightText: Copyright (c) 2023-2024 Orange`

### CODE REVIEW

**Comment by stanlee974** [comment]
> I have edited all occurrence of 
> - `Copyright (c) 2023 Orange` to `Copyright (c) 2024 Orange`
> - `SPDX-FileCopyrightText: Copyright (c) 2022-2023 Orange` to `SPDX-FileCopyrightText: Copyright (c) 2023-2024 Orange`

Thanks. 
Can you adapt `SPDX-FileCopyrightText: Copyright (c) 2023-2024 Orange` to `SPDX-FileCopyrightText: Copyright (c) 2022-2024 Orange` to cover 2022 too ? 
You can commit and push in the same branch 😊

File: packages/assistant-electron/copy-files.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /**
 * Software Name : UUV
-* SPDX-FileCopyrightText: Copyright (c) 2022-2023 Orange
+* SPDX-FileCopyrightText: Copyright (c) 2023-2024 Orange
```
Comment (stanlee974) [inline packages/assistant-electron/copy-files.js:3]: You can set 2022-2024 Here and other SPDX-FileCopyrightText 😉

**Review by stanlee974 — CHANGES_REQUESTED** [review]

File: packages/assistant-electron/copy-files.js (hunk 1 above)
Comment (khaledatia25) [reply packages/assistant-electron/copy-files.js]: I've done it, you can check it.

**Review by stanlee974 — APPROVED** [review]
It's perfect.thank you 😊

**Pull request merged**